quest/api/providers.py

```diff
@@ -22,7 +22,7 @@ def get_providers(expand=None):
             list of all available providers
 
     """
-    providers = load_providers()
+    providers = load_providers(update_cache=True)
```

Comment:
Why did you change this to `update_cache=True`?

Reply:
Got it. Thanks